lib/ansible/cli/galaxy.py

```diff
@@ -110,6 +111,7 @@ def init_parser(self):
         self.add_build_options(collection_parser, parents=[common, force])
         self.add_publish_options(collection_parser, parents=[common])
         self.add_install_options(collection_parser, parents=[common, force])
+        self.add_verify_options(collection_parser, parents=[common])
```

Comment:
Could you use the `collections_path` parents so to save you from having to add the `-p` argument in `add_verify_options()`?
```suggestion
        self.add_verify_options(collection_parser, parents=[common, collections_paths])
```